shoulder/parser/armv8_xml_parser.py

```diff
@@ -54,13 +55,48 @@ def parse_registers(self, path):
                     logger.debug("Register Attributes:")
                     reg = model.Register()
                     self._set_register_name(reg, reg_node)
-                    self._set_register_long_name(reg, reg_node)
-                    self._set_register_access_attributes(reg, reg_node)
-                    self._set_register_access_mechanisms(reg, reg_node)
-                    self._set_register_purpose(reg, reg_node)
-                    self._set_register_size(reg, reg_node)
-                    self._set_register_fields(reg, reg_node)
-                    registers.append(reg)
+                    if "<n>" in reg.name:
+                        array_start_node = reg_node.find("./reg_array/reg_array_start")
+                        array_end_node = reg_node.find("./reg_array/reg_array_end")
+
+                        if array_start_node is None or array_end_node is None:
```

Comment:
Seems like ERRNMISC0 is the only register that matches this criteria. Makes me wonder if you found a bug in the spec?